servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -227,6 +272,22 @@ void RenderForwardClustered::RenderBufferDataForwardClustered::configure(RID p_c
 
 		depth_msaa = RD::get_singleton()->texture_create(tf, RD::TextureView());
 
+		if (view_count == 1) {
+			// just reuse
+			color_views[0] = color;
+			depth_views[0] = depth;
+			color_msaa_views[0] = color_msaa;
+			depth_msaa_views[0] = depth_msaa;
+		} else {
+			// create slices
+			for (uint32_t v = 0; v < view_count; v++) {
+				color_views[v] = RD::get_singleton()->texture_create_shared_from_slice(RD::TextureView(), color, v, 0);
```

Comment:
I'm recreating slices on the color and depth buffers here because the slices are only available outside of the renderer implementation. This we should clean up once we reorganize the renderer logic so we have access to members of our main render buffer.